README.md

```diff
@@ -1,6 +1,9 @@
 # CVPR'21 Continual Learning Challenge Seed
 
-See repo for CVPR'21 Continual Learning Challenge, both for the Supervised Learning and Reinforcement Learning track.
+Seed repo for the [CVPR'21 Continual Learning Challenge](https://eval.ai/web/challenges/challenge-page/829/overview).
+
+This competition is centered around [Sequoia](https://github.com/lebrice/Sequoia), a framework for standardizing and organizing ML research problems, particularly in the field of Continual Learning, both in CRL and CSL.
```

Comment:
I understand that you wanna push Sequoia, but it's already unavoidable for participants and I really don't think this is necessary to advertise even further here.

Reply:
I respectfully disagree!